app/controllers/issue_monitoring_reviews_controller.rb

```diff
@@ -28,6 +28,19 @@ def destroy
     redirect_back fallback_location: complaint_path(issue_id)
   end
 
+  def update
+    @issue_monitoring_review = IssueMonitoringReview.find(params[:id])
+
+    respond_to do |format|
+      if @issue_monitoring_review.update(review_id: review_id, access_token: hses_access_token)
+        format.js { render inline: "location.reload(true);" }
+      else
+        format.js { render inline: "alert('there was an error changing the id of the RAN review');" }
+        # format.js { render "update_errors" }
```

Comment:
[nitpick] please remove this comment!

Reply:
I left it in there as a reminder to put it back in if we do #207 , but I can remove it and the matching TTA one and just leave a comment in the issue, if you prefer!

Reply:
i'd prefer it to be a TODO to make what needs to happen really clear, if you want to leave it in the code.